themes/unl/templates/record/cover.phtml

```diff
@@ -0,0 +1,7 @@
+<? if (!empty($cover) ): ?>
```

Comment:
We may need to come back to this again - this always slowed things down trying to test if a valid image was found